frontend/components/footer/flex/FooterFlex.vue

```diff
@@ -51,7 +51,9 @@
         {{ $t("components.footer._global.powered-by-netlify") }}
       </a>
       <div class="mt-2 text-light-text dark:text-dark-text">
-        {{ $t("components._global.copyright") }}
+        {{
+          $t("components._global.copyright", { year: new Date().getFullYear() })
```

Comment:
I really like this, @lucianmurmurache! 👏 Super cool! Let us know if you come across other places where we can pass some variables to the i18n strings so clean them up, but also keep in mind that word order might need to change in other languages :)

Reply:
@andrewtavis sure I can take a look and create a PR if I find anything. In terms of the word order, if you meant for the change I made here, that should not be an issue, when writing the translation, all we need to do is to ensure that `{ year }` gets placed in the right spot and it should work like a charm!

Reply:
All sounds great, @lucianmurmurache!